### DISCUSSION TITLE
Constantly matching wrong TV Show

### DISCUSSION THREAD

**Discussion by Karmalakas** [Flexget/Flexget#3164] [Q&A]
There's this new(ish) TV Show _See_, but Flexget constantly matches some old show _See No Evil_ episodes. Any way to make it work? Is there a way in config to define the release year to be added to search maybe?

**Comment by paranoidi** [comment]
Set the "exact" option to yes

**Reply by Karmalakas** [reply]
Sadly, didn't help. Still matched same _See no evil_ 😞

**Reply by Karmalakas** [reply]
Hmm.. That's weird. This week it matched correctly. Maybe config didn't reload for some reason, but I thought I did reload it.
I guess it could've been a mistake from my side, but not sure where.
Thanks for the answer

**Reply by Karmalakas** [reply]
Well... It seems it doesn't work after all 🙁 
Now I have TV show _Invasion_ and it matches _Invasion of the Job Snatchers_

**Reply by gazpachoking** [reply]
What does your config look like? The 'exact' option is to prevent exactly these cases.

**Reply by Karmalakas** [reply]
It's quite a huge config, but I can try to clean up if you get lost in it 🙂 Just rename to _yml_
[config.txt](https://github.com/Flexget/Flexget/files/7401050/config.txt)

I really appreciate both of you helping me and hopefully can pinpoint where the problem is
